ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -4611,4 +4611,241 @@ AHU_DFSS_DX2ZC_ECONZ_CO2C_DSP_RCKTM2X_HTZC_CC2XM_RTM_DTM:
   - RTM
   - DTM
   opt_uses:
-  - condensing_fan_current_sensor
\ No newline at end of file
+  - condensing_fan_current_sensor
+  
+
+
+#PAC Unit#  
+AHU_CHWPM_SFSS_DXSC_SFRDM_CPRDM_EVFSC_STC_ZLTM_ZLPM_ZLHM_DXSC_SFRMM_CHWDT_ZTM:
+  description: "AHUs with chilled water temperature monitoring and compressor monitoring with zone temperature and humidity monitoring ."
+  is_canonical: true
+  implements:
+  - AHU
+  - CHWPM
+  - SFSS
+  - DXSC
+  - SFRDM
+  - CPRDM
+  - EVFSC
+  - STC
+  - ZLTM
+  - ZLPM
+  - ZLHM
+  - DXSC
+  - SFRMM
+  - CHWDT
+  - ZTM  
+
+#PAC Unit#  
+AHU_SFSS_DX2ZTC_SFRDM_EVFSC_STC_ZLTM_HRPM2X_LRPM2X_ZHLM_SFRMM_ZTM:
+  description: "AHUs with chilled water temperature monitoring and compressor monitoring with zone temperature and humidity monitoring ."
+  is_canonical: true
+  implements:
+  - AHU
+  - SFSS
+  - DX2ZTC
+  - SFRDM
+  - EVFSC
+  - STC
+  - ZLTM
+  - HRPM2X
+  - LRPM2X 
+  - ZLHM
+  - SFRMM
+  - ZTM  
+
+#AHU Unit#  
+
+AHU_RMM_SFMSC_SFSS_SFVSC_FDPM_STC_ZA_RACO2C_CHWVM_ECONM2X_SAIDC_CHWSC_CO2C_SFRDM_SRC_EPM:
+  description: "AHUs with chilled water temperature monitoring and compressor monitoring with zone temperature and humidity monitoring ."
+  is_canonical: true
+  implements:
+  - AHU
+  - RMM
+  - SFMSC
+  - SFSS
+  - SFVSC
+  - FDPM
+  - STC
+  - ZA
+  - RACO2C
+  - CHWVM
+  - ECONM2X
+  - SAIDC
+  - CHWSC
+  - CO2C
+  - SFRDM
+  - SRC
+  - EPM
+  
+AHU_RMM_SFMSC_SFSS_SFVSC_FDPM_STC_ZA_RACO2C_CHWVM_ECONM2X_SAIDC2X_RAIDC2X_CHWSC_CO2C_SFRDM_SRC_EPM:
+  description: "AHUs with chilled water temperature monitoring and compressor monitoring with zone temperature and humidity monitoring ."
+  is_canonical: true
+  implements:
+  - AHU
+  - RMM
+  - SFMSC
```

Comment:
Why does this device have discrete speed control and VFD speed control? You should not have both on the same device.